javaagent-api/javaagent-api.gradle

```diff
@@ -10,20 +10,15 @@ apply from: "$rootDir/gradle/publish.gradle"
 project.ext.minimumBranchCoverage = 0.0
 project.ext.minimumInstructionCoverage = 0.0
 
-javacc {
-  configs {
-    sanitizer {
-      inputFile = file('src/main/javacc/SqlSanitizer.jj')
-      packageName = 'io.opentelemetry.javaagent.instrumentation.api.db.sanitizer'
-      outputDir = file("${project.buildDir}/generated/javacc/sql")
-    }
-    statmentInfoExtractor {
-      inputFile = file('src/main/javacc/SqlStatementInfoExtractor.jj')
-      packageName = 'io.opentelemetry.javaagent.instrumentation.api.db.sanitizer'
-      outputDir = file("${project.buildDir}/generated/javacc/sql")
-    }
-  }
+def jflexTargetDir = file"${project.buildDir}/generated/jflex/sql"
+def jflexTask = tasks.create("sqlSanitizerJflex", org.xbib.gradle.plugin.JFlexTask) {
```

Comment:
The jflex plugin was supposed to automatically create a task for this, but it's currently broken... I'll create an issue in their project. Meanwhile, a more manual approach still works